src/services/theme/theme.ts

```diff
@@ -45,19 +45,11 @@ import { titles_ams } from '../../themes/eui-amsterdam/global_styling/variables/
  * anything using the `color.` key will remain under `color`
  */
 
-// HELP: For some reason removing this causes a type error:
-// TypeError: 'getOwnPropertyDescriptor' on proxy: trap reported non-configurability for property 'length' which is either non-existent or configurable in the proxy target
-const poles = {
-  ghost: '#FFF',
-  ink: '#000',
-};
-
 /* DEFAULT THEME */
 // TODO: All theme files need to be imported here or else they error out.
 // Creation of the themes shouldn't be restricted to a particular file
 
 export const light = {
```

Comment:
Have you tried moving this actual object creation to a separate file? 

Basically what I was trying to accomplish was to have a theme file for the default theme and one for the Amsterdam theme in their respective theme folders, similar to like the manifest SASS `index.scss` files.

Reply:
Should be no problem. Just moved all of Amsterdam in b2cd915
I'm guessing any problems you saw previously were due to that comment bug, which is now fixed.